client/src/components/Pages/Feedback/index.js

```diff
@@ -40,13 +40,16 @@ class Feedback extends Component {
     const getOdd = (el, idx) => idx % 2 !== 0;
 
     return (
-      <div className="feedbacks-container">
-        <div className="row">
-          <div className="col s12 m6">
-            {feedbacks.filter(getEven).map(this.renderFeedbackItem)}
-          </div>
-          <div className="col s12 m6">
-            {feedbacks.filter(getOdd).map(this.renderFeedbackItem)}
+      <div className="feedbacks-section-container">
+        <h3 className="feedbacks-section-title">Feedback</h3>
```

Comment:
This title should be the same for all the pages. Maybe it makes sense to convert it to a separate component with encapsulated properties an styles. I am not sure yet, so I decided to leave it like this for the moment.